package.json

```diff
@@ -58,12 +58,12 @@
     "@angular/router": "~7.0.3",
     "@angular/upgrade": "~7.0.3",
     "@hmcts/ccd-case-ui-toolkit": "2.62.21-RDM-4335-prerelease-2",
-    "@hmcts/ccpay-web-component": "~2.0.50",
+    "@hmcts/ccpay-web-component": "^3.0.0-beta15",
```

Comment:
I've found some references to `ccpay-web-component` in the toolkit as well. A bit old versions thought, so not sure if those are a leftover or still used. Can you please check whether we need to update there as well?